tasks.py

```diff
@@ -31,11 +33,10 @@ def song_links():
 
 @app.task
 def song_lyrics(url, title):
-    page = requests.get(url, headers={'user-agent': USER_AGENT})
-    content = page.text.replace('<br>', '').encode('utf-8')
-    parsed_content = BeautifulSoup(content, 'html.parser')
-    ringtone = parsed_content.find('div', {'class': 'ringtone'})
-    lyrics = ringtone.find_next_sibling('div').text
+    page = requests.get(url, headers={'user-agent': 'Mozilla/5.0 (compatible; Googlebot/2.1; +http://www.google.com/bot.html)'})
```

Comment:
This should use the `USER_AGENT` or maybe just remove it entirely. I don't think it's needed on this site.